README.md

```diff
@@ -1,4 +1,203 @@
-# 🚧 UNDER CONSTRUCTION 🚧
+# Aragon OSX Plugin Template [![Hardhat][hardhat-badge]][hardhat] [![License: AGPL v3][license-badge]][license]
+
+[hardhat]: https://hardhat.org/
+[hardhat-badge]: https://img.shields.io/badge/Built%20with-Hardhat-FFDB1C.svg
+[license]: https://opensource.org/licenses/AGPL-v3
+[license-badge]: https://img.shields.io/badge/License-AGPL_v3-blue.svg
+
+## Quickstart
+
+After [creating a new repository from this template](https://github.com/new?template_name=osx-plugin-template-hardhat&template_owner=aragon), cloning, and opening it in your IDE, run
+
+```sh
+yarn install && cd packages/contracts && yarn install && yarn build && yarn typechain
```

Comment:
could we not have this as a script in the root json?

Reply:
I leave that to you or @mathewmeconry to decide. One person, either owning this repo or being responsible for our build infrastructure should make this decision and try to find the optimum between maintainability and developer friendliness.